cmd/build-images.go

```diff
@@ -126,5 +134,12 @@ func init() {
 		"Output the result as tar files in the given directory rather than building with docker",
 	)
 
+	buildImagesCmd.PersistentFlags().StringP(
+		"stemcell",
+		"",
+		"",
```

Comment:
I think we can have a default - `suse/fissile-opensuse-stemcell` (i think that's what we settled on, right?)